products/bash/register-tracing.sh

```diff
@@ -14,6 +14,7 @@
 #
 # PARAMETERS:
 #   -n : <namespace> (string), Defaults to "cp4i"
+#   -e : <test_environment> (flag), Defaults to "false"
```

Comment:
This comment should doc the -a parameter

Reply:
done